src/dbnode/encoding/proto/encoder_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	"testing"
 	"time"
 
+	dpb "github.com/golang/protobuf/protoc-gen-go/descriptor"
```

Comment:
Hm this import doesn't belong here yeah?

Linter is definitely getting way more flakey, noticed a lot of these.